android/engine/src/main/java/org/smartregister/fhircore/engine/ui/components/PinView.kt

```diff
@@ -214,44 +214,44 @@ fun PinCell(
   }
 }
 
-@ExperimentalComposeUiApi
-@Preview
+@PreviewExcludeGenerated
```

Comment:
Is it possible to overload the `@Preview` annotation? Instead of creating a new one?, but again it will be confusing and it is easier to forget.

Reply:
Yeah that would definitely be better, but kotlin annotation class doesn't allow to do that

Reply:
I tried make it inherit from @Preview but that doesn't work either